config/locales/models/activity.en.yml

```diff
@@ -97,6 +99,7 @@ en:
         flow: Flow type
         geography: Benefitting recipient geography
         delivery_partner_identifier: Identifier
+        roda_identifier: RODA Identifier
```

Comment:
Should probably be a lowercase `i` - "RODA identifier"

Reply:
Just amended this.